src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -69,6 +69,9 @@ public int Main(string[] args)
                 var optUnlock = c.Option("--unlock",
                     "Creates dependencies file with locked property set to false. Overwrites file if it exists.",
                     CommandOptionType.NoValue);
+                var optSkipRestoreEvents = c.Option("--skip-restore-events",
+                    "Suppress the execution of any scripts defined in project.json",
```

Comment:
Remove "any" from "Suppress the execution of any scripts defined in project.json" and :shipit: